timelock-impl/src/test/java/com/palantir/atlasdb/timelock/lock/watch/LockWatchingServiceImplTest.java

```diff
@@ -77,6 +83,23 @@ public void setup() {
         when(locks.locksHeld()).thenReturn(ImmutableSet.of(heldLocks));
     }
 
+    @Test
+    public void runTaskRunsExclusivelyOnLockLog() {
```

Comment:
I'm not a huge fan of this test, but I'm struggling to come up with a better way to confirm the key property here - that the task is synchronised.